tests/algorithm/scripts/send_test_records.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import requests
+from helpers import dict_to_pii
+
+
+def send_test_records(test_csv, algorithm_name, api_url):
+    df = pd.read_csv(test_csv)
```

Comment:
Is pandas only necessary to read and write csv files?  If so, there is a standard lib for that https://docs.python.org/3/library/csv.html

Reply:
Yeah. I will go ahead and switch it to that default reader. Removing it should make the build faster too.